### DISCUSSION TITLE
Clearing a row from child grid  using inline button

### DISCUSSION THREAD

**Discussion by rhemasystems** [serenity-is/Serenity#6652] [Help] [answered]
I want to be able to use an inline button to remove a record from a child grid inside a master dialog.
I want to clear the red circled row  that was added by the **Add Item**  button using inline button without opening it to click on the  **Delete** button to clear it from the grid
Kindly assist
![image](https://user-images.githubusercontent.com/50826372/212082903-01dc7b72-e540-4f35-8887-99f4a1811bca.png)

**Comment by arunkumar12712** [comment, marked as answer]
protected enableDeleteColumn() {
            return true;
        }

**Reply by rhemasystems** [reply]
Thanks arunkumar12712, but where do I place this
I realised when I access the child page directly the inline button shows but when I access it from the master dialog page it doesnt show the inline button

Direct page
![image](https://user-images.githubusercontent.com/50826372/212142662-4e9db13c-9413-47ff-8543-16ac841a51b5.png)


From Master Dialog

![image](https://user-images.githubusercontent.com/50826372/212142543-490d914e-8678-4a8a-8a36-a3fec6bd41f2.png)

**Reply by arunkumar12712** [reply]
Try 
add code in  LineItemseditor 
Like example 
            export class ItemEditor extends Common.GridEditorBase < ItemRow >

*  Tranform all t4
